dist/style.css

```diff
@@ -59,3 +55,288 @@ body {
   -ms-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+
+/* container css for most high level containers  */
+.container-parent {
+  height: 100%;
+  position: relative;
+}
+
+.container-header {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+}
+
+.container-navSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-bottom: 4%;
+}
+
+.container-nav {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 5%;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.container-moto {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  width: 100%;
+  justify-content: left;
+}
+
+/* effects for hover on buttons */
+.menu {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  background-color: white;
+  text-align: center;
+  padding: 2%;
+  border-radius: 25px;
+  border: 2px solid #A9A9A9;
+  width: 25%;
+}
+
+.menu:hover {
+  background-color: #662A61;
+  color: white;
+}
+
+a {
+  text-decoration: none;
+}
+
+.fas:hover {
+  background-color: #562554;
+  color: white;
+}
+
+p {
+  margin: 0px;
+  margin-left: 7.5%;
+  margin-right: 10.5%;
+}
+
+fas {
+  height: 0%;
+}
+
+/* logo resizing  */
+.container-logo {
+  max-width: 65%;
+  height: auto;
+}
+
+#mainlogo {
+  max-width: 65%;
+  height: auto;
+  padding-right: -25%;
+}
+
+
+/* main content in split boxes  */
+.container-body {
+  margin-bottom: 20px;
+}
+
+.content {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  border: 5px solid #562554;
+  border-radius: 25px;
+  background-color: white;
+}
+
+.content-left {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+}
+
+.content-right {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+  text-align: center;
+}
+
+/* form styling  */
+input {
+  border-radius: 25px;
+  width: 100%;
+  background-color: #336D61;
+  color: white;
+  padding: 6.5px;
+  margin-top: 4px;
+  margin-bottom: 1.5px;
+}
+
+::placeholder {
+  color: white;
+  opacity: 1;
+}
+
+.disclaimer {
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 75%;
+  color: rgb(88, 87, 87);
+  text-align: left;
+  padding-right: 5px;
+}
+
+#msg {
+  height: 200px;
+  border-radius: 25px;
+  width: 100%;
+  background-color: #336D61;
+  color: white;
+  margin-bottom: 1.5px;
+  margin-top: 6px;
+  padding-left: 6.5px;
+  padding-top: 16.5px;
+}
+
+/* disclaimer and send button  */
+.container-send {
```

Comment:
Let's reduce some of the margin on this `<p>` tag. It looks like it is inheriting a lot of margin from the `p` selector you've defined on Line 125. Remember, CSS = Cascading Style Sheets. Styles cascade from the top down, so if you don't overwrite that style here, you're going to inherit all the extra margin, which is why this section currently looks so narrow.

Reply:
changed it from a `<p>` tag to a `<span>` tag. Margin was automatically removed.